### DISCUSSION TITLE
RTLCSS with yarn prod and SCSS

### DISCUSSION THREAD

**Discussion by DenyaOver** [twbs/bootstrap#38106] [Q&A]
Hi guys, I am using scss type files for the project, and I compile them with yarn encore prod, and when I writing example from the Bootstrap documentation #{/* rtl:600 */} into my scss file, for some reason browser see this like a standart comment.
What could be the reason?

**Comment by ffoodd** [comment]
Hi there,

Those are standard comments, that's why. [RTLCSS](https://rtlcss.com/) uses them to output the RTL version.

If you want to drop those comments, you may do as Bootstrap currently does:
1. compile a non-minified CSS file, with comments — eg. `bootstrap.css`;
2. compile a minified, compressed CSS file without comments — eg. `bootstrap.min.css`;
3. run RTLCSS on the first file to get `bootstrap.rtl.css`, and optionally minify it on the go to `bootstrap.rtl.min.css` if you don't need the unminified version.

That's how RTLCSS works and how we do it at the moment in Bootstrap. Does it help?
